spec/machine/MachineExports.thy

```diff
@@ -67,32 +67,38 @@ requalify_consts
   timerPrecision
   max_time
   getCurrentTime_buffer
+  \<mu>s_in_ms
 
 requalify_facts
-  us_to_ticks_mono
-  us_to_ticks_zero
-  us_to_ticks_nonzero
-  kernelWCET_ticks_no_overflow
   MAX_PERIOD_US_def
   MAX_PERIOD_def
-  us_to_ticks_mult
-  getCurrentTime_buffer_no_overflow
   kernelWCET_ticks_def
   replicate_no_overflow
-  getCurrentTime_buffer_nonzero
   getCurrentTime_buffer_nonzero'
   getCurrentTime_buffer_no_overflow'
   getCurrentTime_buffer_no_overflow'_stronger
   getCurrentTime_buffer_minus
   getCurrentTime_buffer_minus'
   MAX_PERIOD_mult
-  MAX_PERIOD_mult'
+  ticks_per_timer_unit_non_zero
+  MIN_BUDGET_bound
+  getCurrentTime_buffer_bound
+  kernelWCET_pos'
+  MIN_BUDGET_pos'
+  domain_time_pos
+  getCurrentTime_buffer_pos
+  us_to_ticks_def
```

Comment:
Do we need to export the definition of `us_to_ticks`? It is not meant to be available in the generic context, right?

Reply:
Nope, removed